python/CMakeLists.txt

```diff
@@ -1,22 +1,6 @@
-if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
-  # pybind11 logic for setting up a debug build when both a debug and release
-  # python interpreter are present in the system seems to be pretty much broken.
-  # This works around the issue.
-  set(PYTHON_LIBRARIES "${PYTHON_DEBUG_LIBRARIES}")
-endif()
-
-
 if(USE_SYSTEM_PATHS_FOR_PYTHON_INSTALLATION)
-  if(${CMAKE_VERSION} VERSION_LESS "3.12.0")
-    execute_process(
```

Comment:
I think we should revert this. We can be sure Jammy has CMake > 3.12, but since the whole project sets the minimum version to 3.10, we can't assume that this is being built by a newer version of CMake.

Reply:
All changes related to CMakeLists were removed in 47e82ba